dts/bindings/pinctrl/raspberrypi,pico-pinctrl.yaml

```diff
@@ -121,3 +121,17 @@ child-binding:
         description: |
           The slew rate of a pin. 0 corresponds to slow, and 1 corresponds to fast.
           The default value is 0 (slow), as this is the power on reset value.
+      oe-override:
+        type: int
+        enum:
+          - 0
+          - 1
+          - 2
+          - 3
+        default: 0
+        description: |
+          Override output-enable for a pin.
+          0 - drive output enable from selected peripheral signal (default)
+          1 - drive output enable from inverse of selected peripheral signal.
+          2 - disable output
```

Comment:
From the datasheet 2/3 look like active low / active high, is that correct?

Reply:
I'm not sure what you mean by active low or high. My reading is that setting 2 or 3 means "ignore whether peripheral is sending, behave as if you are getting a low (in the case of 2) or high (in the case of 3) signal".

In the case of OEOVER (the subject of this PR), this means either disabling the output, or enabling the output. 

Source: Rp2040 datasheet, Page 247 (IO BANK0: GPIO0_CTRL, ...)

Reply:
Sorry, I was reading the wrong field in the datasheet.